sdk/python/kfp/containers/_component_builder.py

```diff
@@ -177,7 +177,7 @@ def build_python_component(component_func, target_image, base_image=None, depend
     staging_gcs_path (str): GCS blob that can store temporary build files
     target_image (str): target image path
     timeout (int): the timeout for the image build(in secs), default is 600 seconds
-    namespace (str): the namespace within which to run the kubernetes kaniko job, default is "kubeflow"
```

Comment:
Would you mind adding a little more comments that the build_python_component internally will find the current namespace if it is run in the GKE cluster.

Reply:
good point ! absolutely.